{
  "title": "\"Exception\" should not be caught when not required by called methods",
  "type": "BUG",
  "status": "ready",
  "remediation": {
    "func": "Constant\/Issue",
    "constantCost": "15min"
  },
  "tags": [
    "cwe",
    "error-handling",
    "bug"
  ],
  "standards": [
    "CWE"
  ],
  "defaultSeverity": "Major"
}
